client/src/css/stylesheet.scss

```diff
@@ -0,0 +1,211 @@
+@font-face {
+  font-family: "Open Sans";
+  src: url("../fonts/OpenSans-Regular.ttf") format("truetype");
```

Comment:
Why the relative URL to a parent directory? Where are you planning on serving the files from?